### DISCUSSION TITLE
Walrus | 线上签署合同并保存在链上的平台

### DISCUSSION THREAD

**Discussion by billaGitHub2016** [creatorsand/.github#117] [Ideas]
一句话 IDEA：
可以在线签署合同，并且把合同文件保存在链上的平台。

我的角色：
合约&前端开发

其他角色需求（不包含发起人）：
暂时不需要，开发时间较少，项目可能会难产。

社群昵称：
billa

个人简介：
多年前端开发经验，目前在学习web3相关技术。

产品是给谁用的？
目标客户是需要在线签署合同的公司或个人，签署后的合同文件会保存在walrus，签署方会得到nft作为记录凭证。

其他产品有什么痛点，用户为什么要选择你的产品？
目前有较多的在线签约场景，大多数人都不会保存签署过的合同，或者不知道哪里可以查询到签署过的合同。
这个平台使用一个公开透明的方式保存合同，利用链上数据不可修改的特性，保障签署方的利益。

**Comment by billaGitHub2016** [comment]
代码仓库：https://github.com/billaGitHub2016/walrus-sign-contract/tree/main/src

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
